tenacity/retry.py

```diff
@@ -111,6 +112,61 @@ def __call__(self, attempt):
             return not self.predicate(attempt.result())
 
 
+class retry_if_exception_message(retry_base):
+    """Retries if an exception message equals or matches."""
+
+    def __init__(self, message=None, match=None, require_exception=False):
+        if message and match:
+            raise TypeError(
+                "{}() takes either 'message' or 'match', not both".format(
+                    self.__class__.__name__))
+        self.predicate = self._predicate_factory(message, match)
+        self.require_exception = require_exception
+
+    def _predicate_factory(self, message, match):
+        if message:
+            def message_fnc(exception_message):
+                return message == exception_message
+            return message_fnc
+        elif match:
+            prog = re.compile(match)
+
+            def match_fnc(exception_message):
+                return prog.match(str(exception_message))
+            return match_fnc
+        raise TypeError(
+            "{}() missing 1 required argument 'message' or 'match'".format(
+                self.__class__.__name__))
+
+    def _no_failure(self):
```

Comment:
`@staticmethod`

Not sure it's worth having a method, just an attribute would be enough.